### DISCUSSION TITLE
How bullmq manages connections ?

### DISCUSSION THREAD

**Discussion by aqibamir** [taskforcesh/bullmq#1678] [General]
Whenever I'm running a worker and it process a new job it display the warning 

```
It is highly recommended to use a minimum Redis version of 6.2.0
    Current: 6.0.9
```
So is it opening a new connection whenever it is processing a new job ?

**Comment by manast** [comment]
No, that warning is displayed once, if it is displayed more is because you are instantiating new workers.
